makefile

```diff
@@ -54,15 +50,21 @@ COMMON_CFLAGS += -DWITH_IPF
 LIBS += $(MINGW_PATH)/bin/$(CAPSIPFDLL)
 endif
 else
+prefix = /usr/local
+TARGET = cap32
+TEST_TARGET = test_runner
 IPATHS = -Isrc/ -Isrc/gui/includes `freetype-config --cflags` `sdl-config --cflags` `pkg-config --cflags libpng`
 LIBS = `sdl-config --libs` -lz `freetype-config --libs` `pkg-config --libs libpng`
+CXX ?= g++
 ifdef WITH_IPF
 COMMON_CFLAGS += -DWITH_IPF
 LIBS += -lcapsimage
 endif
-ifndef CXX
-CXX = g++
 endif
+
+ifndef RELEASE
```

Comment:
No need to fix in this PR, but would be nice to find a way to integrate it in AppVeyor build when building on a tag.